src/css/styles.css

```diff
@@ -40,11 +40,26 @@ body {
   display: flex;
   align-items: flex-end;
 }
+
 .sidebar-form {
   margin: 10px;
   height: 300px;
 }
 
+form > label {
+  font-size: 15px;
+  font-weight: bold;
+  grid-column: 1;
+}
+
+.trip-form {
+  display: grid;
+  grid-template-columns: 100px, 100px;
+  grid-template-rows: 50px, 50px, 50px, 50px;
+  /* grid-gap: 5px;
```

Comment:
Whenever you have a minute to refactor to remove these comments, that would clean up this file nicely